app/views/users/_member_card.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="col-lg-6 col-sm-12">
+<div class="col-lg-6 col-sm-12 column--cards">
```

Comment:
you are applying flex and flex direction to an element which only has 1 child so I'd say that we can remove this class